### ISSUE TITLE
quat_feat implementation

### ISSUE THREAD

**Issue by gnool** [ahmedosman/STAR#33]
@ahmedosman 
in the quat_feat function, in the line
`quat = torch.cat([v_sin * normalized,v_cos-1], dim=1)`

is the `-1` intentional? The comment says it should return a normalized quaternion but this `-1` seems to make it non-normalized.
